readme.md

```diff
@@ -100,6 +100,13 @@ Default: `true`
 
 Allow symbolic links to match if they point to the chosen path type.
 
+##### stopAt
+
+Type: `string`\
+Default: `path.parse(cwd).root`
+
+The absolute path to the directory to stop the search before reaching root if there were no matches before the `stopAt` directory.
```

Comment:
Why does it have to be an *absolute* path? I think you can just `path.resolve` the input path you get from the user.